langchain/src/storage/tests/convex/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This sets up an entire subproject?

Reply:
This is needed unfortunately by `npx convex dev`. But it doesn't lead to any installed node_modules.